sdk/formrecognizer/azure-ai-formrecognizer/README.md

```diff
@@ -119,49 +130,53 @@ Once completed, set the values of the client ID, tenant ID, and client secret of
 
 ```python
 from azure.identity import DefaultAzureCredential
-from azure.ai.formrecognizer import FormRecognizerClient
+from azure.ai.formrecognizer import DocumentAnalysisClient
 credential = DefaultAzureCredential()
 
-form_recognizer_client = FormRecognizerClient(
+document_analysis_client = DocumentAnalysisClient(
     endpoint="https://<my-custom-subdomain>.cognitiveservices.azure.com/",
     credential=credential
 )
 ```
 
 ## Key concepts
 
-### FormRecognizerClient
-`FormRecognizerClient` provides operations for:
+### DocumentAnalysisClient
+`DocumentAnalysisClient` provides operations for analyzing input documents using custom and prebuilt models through the `begin_analyze_document` and `begin_analyze_document_from_url` APIs.
+Use the `model` parameter to select the type of model for analysis.
+
+|Model | Features
```

Comment:
Tiny nit: does the space after model need to exist?

Reply:
Same question for the rows below.

Reply:
I can remove, not sure if it affects rendering either way